test/index.js

```diff
@@ -74,6 +78,7 @@ describe('Matrix operations', () => {
     
     it('should find the product of two matrices', () => {
        assert.deepEqual(mat1.prod(mat2), [[6, 6, 6], [15, 15, 15], [24, 24, 24]]); 
+       assert.deepEqual(mat3.prod(mat4), [[58, 64], [139, 154]]);
```

Comment:
Can you add a case of multiplication between:
`[[7, 8],[9, 10],[11, 12]]` and `[[1, 2, 3], [4, 5, 6]]`. It is `2 X 3` * `3 X 2`. You added `3 X 2` * `2 X 3`